fe/fe-core/src/main/java/com/starrocks/sql/optimizer/transformer/RelationTransformer.java

```diff
@@ -388,29 +387,26 @@ public LogicalPlan visitTable(TableRelation node, ExpressionMapping context) {
 
     @Override
     public LogicalPlan visitCTE(CTERelation node, ExpressionMapping context) {
-        if (session.getSessionVariable().isCboCteReuse()) {
```

Comment:
Will always enable CteReuse?

Reply:
Analyze will follow CTE process, will check the flag in rule rewrite phase, if disable CTEReuse, will inline all CTE query